app/components/appMainModule/committeePageModule/committeePageRoutes.js

```diff
@@ -8,19 +8,23 @@
 'use strict';
 
 function committeePageRoutes($stateProvider) {
-//module.exports = function($stateProvider) {
   $stateProvider.state({
     name: 'appMain.committee',
-    // name: 'appMain.locality.committee',
     abstract: true,
-    url: '^/committee/:committee_id',
+    url: '^/committee/:filer_id',
     template: '<ui-view></ui-view>',
     controller: 'committeePageController',
     resolve: {
-      committee: function($stateParams, disclosureApi) {
-        return disclosureApi.committee.get({
-          committee_id: $stateParams.committee_id
+      committee: function($stateParams, static_api) {
+        return static_api.committee.get({
+          filer_id: $stateParams.filer_id
+        });
+      },
+      contributions: function($stateParams, static_api) {
+        var apiCall = static_api.committee.contributions({
+          filer_id: $stateParams.filer_id
         });
+        return apiCall.$promise;
```

Comment:
That's a fair point. If we scrap ngResource, I'd prefer to still use promises and to keep the api logic tucked away in some kind of service. I only used ngResource because it's in [angular core](https://docs.angularjs.org/api/ngResource) and thought it was the defacto.